packages/code/src/Code/Code.spec.tsx

```diff
@@ -163,6 +151,7 @@ describe('packages/Code', () => {
                 spy.mockReturnValue(true);
                 return renderCode({
                   isLoading: true,
+                  // @ts-expect-error
                   copyable: false,
```

Comment:
why are we passing in copyable here if it's removed?

Reply:
i think the point is to test that it would error, but i'll rm.